sdk/containerregistry/azure-containerregistry/azure/containerregistry/_anonymous_exchange_client.py

```diff
@@ -36,25 +36,25 @@ def __init__(self, endpoint, **kwargs):  # pylint: disable=missing-client-constr
         )
 
     def get_acr_access_token(self, challenge, **kwargs):
-        # type: (str, Dict[str, Any]) -> str
+        # type: (str, Any) -> str
         parsed_challenge = _parse_challenge(challenge)
         parsed_challenge["grant_type"] = TokenGrantType.PASSWORD
         return self.exchange_refresh_token_for_access_token(
-            None,
+            "",
```

Comment:
This seems suspicious - what was the reason?

Reply:
The value will pass to self._client.authentication.exchange_acr_refresh_token_for_acr_access_token() which is required for this parameter. I checked other languages, like in .Net is using empty string.